docker-compose.yml

```diff
@@ -70,12 +104,16 @@ services:
     image: mongo-express
     restart: always
     ports:
-      - 8000:8081
+      - 8001:8081
```

Comment:
Keeping mongo-express for oauth2. Oauth2 will be deleted in the future as unused

Reply:
jira ticket created?